app/src/main/res/layout/fragment_mentorship_tasks.xml

```diff
@@ -66,16 +66,18 @@
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toBottomOf="@+id/view" />
 
-    <android.support.v7.widget.RecyclerView
+    <androidx.recyclerview.widget.RecyclerView
         android:id="@+id/rvAchievements"
-        android:layout_width="wrap_content"
+        android:layout_width="0dp"
         android:layout_height="wrap_content"
-        app:layout_constraintVertical_weight="0.5"
-        android:layout_marginStart="16dp"
-        android:layout_marginEnd="16dp"
-        app:layout_constraintTop_toBottomOf="@id/tvAchievements"/>
+        android:layout_marginStart="8dp"
+        android:layout_marginEnd="8dp"
```

Comment:
this is a change in the UI right? Does it need to be here? If you think it is, be explicit about this o nthe PR description, show the changes visually (through an image for example)

Reply:
Actually I ran into a similar problem here as with my last PR. This particular recyclerview in question was not constrained properly, and this made the lint check fail. So I added these contraints myself. I'm not sure why these changes were merged without proper constraints in the first place.

Reply:
Ok, I asked you to add this piece of change to the PR description :) I'll add it